src/CoreBundle/Model/Metadata.php

```diff
@@ -39,6 +39,11 @@ class Metadata implements MetadataInterface
      */
     protected $image;
 
+    /**
+     * @var boolean
+     */
+    private $isImageAvailable;
```

Comment:
This class is deprecated, we shouldn't add any features to this class anymore

Reply:
In this case admins that still use it won't benefit from mosaic background... Not a problem? Should I add a deprecation notice about it or just close PR?